podman_tutorial.md

```diff
@@ -0,0 +1,156 @@
+commands with Podman.
+
+## Install Podman on Fedora
+Many of the basic components to run Podman are readily available from the Fedora RPM repositories; the only
+exception is Podman itself.  In this section, we will help you install all the runtime and build dependencies
+for Podman until an RPM becomes available.
+
+### Installing build and runtime dependencies
+```
+# sudo dnf install -y git runc libassuan-devel golang golang-github-cpuguy83-go-md2man glibc-static \
+                                    gpgme-devel glib2-devel device-mapper-devel libseccomp-devel \
+                                    atomic-registries iptables skopeo-containers containernetworking-cni
+```
+### Building and installing podman
+```
+# git clone https://github.com/projectatomic/libpod/ ~/src/github.com/projectatomic/libpod
+# cd !$ (or cd src/github.com/projectatomic/libpod
+# make
+# sudo make install PREFIX=/usr
+```
+
+<!-- (
+## Install podman on Ubuntu
+
+The default Ubuntu cloud image size will not allow for the following exercise to be done without increasing its
+capacity.  Be sure to add at least 5GB to the image. Instructions to do this are outside the scope of this
+tutorial.
+
+```
+# sudo apt-get update
+# sudo apt-get install libdevmapper-dev libglib2.0-dev libgpgme11-dev golang libseccomp-dev \
+                        go-md2man libprotobuf-dev libprotobuf-c0-dev libseccomp-dev
+# mkdir -p ~/src/github.com/projectatomic/
+# cd ~/src/github.com/projectatomic/
+# git clone https://github.com/projectatomic/libpod/
+#
+# cd libpod
+# make
+# sudo make install PREFIX=/usr
+# sudo mkdir -p /etc/containers
+# sudo bash -c 'cat <<EOF > /etc/containers/registries.conf
+registries.search
+registries = ['docker.io', 'registry.fedoraproject.org']
+EOF
+'
+# sudo bash -c 'cat <<EOF > /etc/containers/policy.json
+ {
+     "default": [
+         {
+             "type": "insecureAcceptAnything"
+         }
+     ],
+     "transports":
+         {
+             "docker-daemon":
+                 {
+                     "": [{"type":"insecureAcceptAnything"}]
+                 }
+         }
+ }
+ EOF
+'
+# git clone https://github.com/containernetworking/plugins.git ~/src/github.com/containernetworking/plugins
+# cd ~/src/github.com/containernetworking/plugins
+# ./build
+# sudo mkdir -p /usr/libexec/cni
+# sudo cp bin/* /usr/libexec/cni
+# git clone https://github.com/opencontainers/runc.git ~/src/github.com/opencontainers/runc
+# GOPATH=~/ make static BUILDTAGS="seccomp selinux"
+# sudo cp runc /usr/bin/runc
+# sudo mkdir -p /usr/local/libexec
+# sudo ln -s /usr/libexec/crio /usr/local/libexec/
+```
+
+) -->
+## Familiarizing yourself with Podman
+
+### Running a sample container
+This sample container will run a very basic httpd server that serves only its index
+page.
+```
+# sudo podman run -dt -e HTTPD_VAR_RUN=/var/run/httpd -e HTTPD_MAIN_CONF_D_PATH=/etc/httpd/conf.d \
```

Comment:
This is failing for me unless I add --network=host.  Should that be added to this sample?

Reply:
you cannot do this tutorial while connected to the RH vpn

Reply:
FWIW, I was on the RedHat  network and was not using the VPN.  Also the VM that I spun up was local to my PC.  Your call.